README.md

```diff
@@ -2,4 +2,6 @@
 
 Sorting filtering and iterating through data using streams
 
-Shows how to do in streams, but also compares it to the pre-streams approach
```

Comment:
Can you update this to say something like:

Shows you how you can use a "functional programming" approach to iterating through Collections with the Java language.